app/errors.py

```diff
@@ -0,0 +1,14 @@
+class NotWearingMaskError(BaseException):
```

Comment:
If you don't have a direct need to inherit from BaseException, just use Exception. Is [recommended in the Python Doc](https://docs.python.org/3/library/exceptions.html#Exception)